packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -41,6 +42,10 @@ typedef SelectionChangedCallback = void Function(TextSelection selection, Select
 /// Signature for the callback that reports the app private command results.
 typedef AppPrivateCommandCallback = void Function(String, Map<String, dynamic>);
 
+/// Signature for the generator function that produces an InlineSpan for replacement
```

Comment:
InlineSpan and TextEditingInlineSpanReplacement could be in square brackets to link them in the docs